turbo.json

```diff
@@ -1,13 +1,13 @@
 {
   "$schema": "https://turbo.build/schema.json",
+  "globalEnv": ["AUTH_SECRET", "DATABASE_URL"],
```

Comment:
Couldn't help but notice that you had some environment variables that were being considered. I didn't know which precise tasks need them so I didn't specify them more-precisely, but I suspect that this should be:

```
"@webstudio-is/builder#build": {
  "env": ["AUTH_SECRET", "DATABASE_URL"]
}
```